db/migrations/20221226203000_db_index.up.sql

```diff
@@ -0,0 +1,59 @@
+CREATE INDEX IF NOT EXISTS index_invoices_on_user_id ON invoices(user_id);
+
+--migration:splitt
+
+CREATE INDEX IF NOT EXISTS index_invoices_on_payment_request ON invoices(payment_request);
+
+--migration:split
+
+CREATE INDEX IF NOT EXISTS index_invoices_on_type ON invoices(type);
+
+--migration:split
+
+--CREATE INDEX IF NOT EXISTS index_invoices_on_state ON invoices(state);
+
+--migration:split
+
+CREATE INDEX IF NOT EXISTS index_invoices_on_r_hash ON invoices(r_hash);
+
+--migration:split
+
+CREATE INDEX IF NOT EXISTS index_invoices_on_created_at ON invoices(created_at);
+
+--migration:split
+
+CREATE INDEX IF NOT EXISTS index_invoices_on_expires_at ON invoices(expires_at)
+
+--migration:split
+
+CREATE INDEX IF NOT EXISTS index_invoices_on_settled_at ON invoices(settled_at)
+
+--migration:split
+
+CREATE INDEX IF NOT EXISTS index_invoices_on_type_and_r_hash_and_state ON invoices(type, r_hash, state);
```

Comment:
This one should speed up query 1 from here:

https://github.com/getAlby/meta/issues/290